pkg/lib/operation/project/local/template/use/operation.go

```diff
@@ -131,6 +131,11 @@ func PrepareTemplate(ctx context.Context, d dependencies, o ExtendedOptions) (pl
 	// Load template state
 	plan.templateState, err = o.Template.LoadState(o.TemplateCtx, LoadTemplateOptions(), d)
 	if err != nil {
+		var invalidState loadState.InvalidLocalStateError
+		if errors.As(err, &invalidState) {
+			// Strip confusing  local state is invalid:" error envelope.
+			return nil, errors.PrefixError(invalidState.Unwrap(), "template definition is not valid")
+		}
```

Comment:
Ked som to fixol, tak som si vsimol, ze to generuje `project local state is invalid`, co je sposobene tym, ze nacitanie projektu aj sablony z disku, pouziva rovnaky kod. 

Takze som fixol aj tuto hlasku.